### DISCUSSION TITLE
When I embed the script to my website, it shows Error: Failed to fetch

### DISCUSSION THREAD

**Discussion by wanqiangliang** [FlowiseAI/Flowise#2243] [Q&A]
Error: Failed to fetch

**Comment by toi500** [comment]
Where is the flowise instance deployed? Cant be in localhost if you embed on a website.

**Reply by wanqiangliang** [reply]
@toi500 Thank you for your info. It's deployed on a machine with a public IP address, and it is accessible from the outside.

**Reply by noblerboy2004** [reply]
same problem with my

**Reply by toi500** [reply]
> same problem with my

Here is a potential fix: https://docs.flowiseai.com/using-flowise/embed#cors

**Reply by noblerboy2004** [reply]
> > same problem with my
> 
> Here is a potential fix: https://docs.flowiseai.com/using-flowise/embed#cors

Thank you for your help. It fixed my problem :)

**Reply by sgalili** [reply]
i'm still having the same problem "Error: Failed to Fetch". my flowise is deployed on my server. 
I can access it from here: http://167.71.129.40:3000/chatbot/8801a37a-88bf-499c-a16d-7d4ea1cc56ae
but cannot access it from the domain: chat.pi1000.co.il where I embedded it

**Reply by gerald-CL** [reply]
> i'm still having the same problem "Error: Failed to Fetch". my flowise is deployed on my server. I can access it from here: http://167.71.129.40:3000/chatbot/8801a37a-88bf-499c-a16d-7d4ea1cc56ae but cannot access it from the domain: chat.pi1000.co.il where I embedded it

Hi, did you manage to solve it? I'm having the same problem...

**Comment by toi500** [comment]
@siddiqss Set your CORS envs like this:

CORS_ORIGINS="https://chat.pi1000.co.il"
EMBEDDING_ORIGINS="https://chat.pi1000.co.il"

**Reply by gerald-CL** [reply]
Hi, when I use the command "EMBEDDING_ORIGINS", it says there's an issue with the command and apparently it's not available.
